config/environments/production.rb

```diff
@@ -55,15 +52,8 @@
   config.active_support.deprecation = :notify
 
   # Logging
-  config.log_tags = [:request_id] # Prepend all log lines with the following tags.
   config.log_level = Settings.log_level
-
-  if Settings.logstash.host && Settings.logstash.port
-    config.logger = LogStashLogger.new(Settings.logstash.to_h)
-  else
-    config.rails_semantic_logger.format = :json
-    config.semantic_logger.add_appender(io: STDOUT, level: config.log_level, formatter: config.rails_semantic_logger.format)
-    config.logger = ActiveSupport::Logger.new(STDOUT)
-    config.logger.info("Application logging to STDOUT")
-  end
+  
+  # Prepend all log lines with the following tags.
+  config.log_tags = [:request_id]
```

Comment:
Looking at the PR description and commit msg I'm not entirely sure why a lot of the logger config has been removed or moved entirely?

Reply:
logstash gem is not required, and we use semantic_logger to format the logs.
Moving semantic logger config into its own initializer file.